### ISSUE TITLE
[tweak] Please allow the media file names to have spaces. Or document that its not allowed

### ISSUE THREAD

**Issue by PrinceOfAmber** [Vazkii/Ambience#17]
After much time fighting and reading the docs and watching your video and testing out the runescape pack, I FINALLY randomly assumed taht the .mp3 file names shouldnt have spaces.  Even though in the config somewhere it says you can replace spaces with the plus sign (+).

If you cant support this, i get it. but please document it  on the curseforge page.  Thanks!  Love the mod.

**Comment by P3rf3ctXZer0** [comment]
Yeah; your not going to get that help I am afraid @PrinceOfAmber I have been on @Vazkii for a few things. Aside from that unless the mod just doesn't work he doesn't tend to respond. He may be busy with life.

**Comment by liam4** [comment]
1. This mod hasn't been looked at for months. The fact that it's not going to get implemented (by Vazkii anyways) seems kind of obvious at this point :P
2. Or maybe they've just moved on from the mod? It sure looks like they're still an [active part](https://twitter.com/vazkii) of The Internet Community..

OTOH if anybody's interested in making a pull request that implements this, that might have more of a chance of getting pulled than writing the code to fix this. (But don't quote me on that please.)
